cage/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import SearchResutlsView, HomepageView
```

Comment:
Is `SearchResutlsView` a typo? Rather than `SearchResultsView`? 

Have you got these views implemented? I notice they're class-based views, are they from a youtube vid?

Reply:
No not yet. Yeah I think I need to look at it again find another example of doing a search bar.

Reply:
Class-based views are useful, I can help you with that next week. Just check your spelling of the view and once you push that change I can approve this.